tracing/src/primitives.rs

```diff
@@ -0,0 +1,13 @@
+use bson::oid::ObjectId;
+use bson::Binary;
+use serde::{Deserialize, Serialize};
+
+// The storage format in MongoDB.
+#[derive(Serialize, Deserialize)]
+pub struct SpanChunk {
```

Comment:
maybe not use "Chunk" here to not confuse with our concept of block chunks?

Reply:
Yeah I guess RawTrace is better.